spec/nosto.load.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { vi, describe, expect, it } from "vitest"
+import { render } from "@testing-library/react"
+import { NostoProvider, NostoHome } from "../src/index"
+
+describe("Nosto client script loading", () => {
+  it("is stopped via provider when the script is loaded externally", async () => {
+    window.nosto = {
+      reload: vi.fn()
+    }
+
+    render(
+      <NostoProvider account="shopify-11368366139" loadScript={false}>
```

Comment:
is loadScript already available?

Reply:
True, I thought it was already available in master.

Reply:
I'll add the test case as it.todo